src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -205,10 +206,10 @@ public string GetLanguage1Name(string inLanguage)
 
 		private string GetLanguage1Name_NoCache(string inLanguage)
 		{
-			Iso639LanguageCode exactLanguageMatch = _lookupIsoCode.GetExactLanguageMatch(Language1Iso639Code);
-			if (exactLanguageMatch == null)
+			var exactLanguageMatch = _lookupIsoCode.GetBestLanguageName(Language1Iso639Code);
+			if (exactLanguageMatch == Language1Iso639Code) // no useful name found
```

Comment:
Actually I hadn't thought of a language whose full name matches its ISO. I was wondering whether we might miss finding a name for a macro language that is NOT sufficiently similar to its ISO.
I've made a card with the suggestion that we use the Try pattern so as to return a boolean indicating whether we got a name.